app/grandchallenge/workstation_configs/models.py

```diff
@@ -124,32 +124,24 @@
 
 
 class WorkstationConfig(TitleSlugDescriptionModel, UUIDModel):
-    ORIENTATION_AXIAL = "A"
-    ORIENTATION_CORONAL = "C"
-    ORIENTATION_SAGITTAL = "S"
-
-    ORIENTATION_CHOICES = (
-        (ORIENTATION_AXIAL, "Axial"),
-        (ORIENTATION_CORONAL, "Coronal"),
-        (ORIENTATION_SAGITTAL, "Sagittal"),
-    )
+    class Orientation(models.TextChoices):
+        AXIAL = "A", "Axial"
+        CORONAL = "C", "Coronal"
+        SAGITTAL = "S", "Sagittal"
 
-    SLAB_RENDER_METHOD_MAXIMUM = "MAX"
-    SLAB_RENDER_METHOD_MINIMUM = "MIN"
-    SLAB_RENDER_METHOD_AVERAGE = "AVG"
+    class SlabRenderMethod(models.TextChoices):
+        MAXIMUM = "MAX", "Maximum"
+        MINIMUM = "MIN", "Minimum"
+        AVERAGE = "AVG", "Average"
 
-    SLAB_RENDER_METHOD_CHOICES = (
-        (SLAB_RENDER_METHOD_MAXIMUM, "Maximum"),
-        (SLAB_RENDER_METHOD_MINIMUM, "Minimum"),
-        (SLAB_RENDER_METHOD_AVERAGE, "Average"),
-    )
+    class ImageContext(models.TextChoices):
+        PATHOLOGY = "PATH", "Pathology"
+        OPHTHAMOLOGY = "OPHTH", "Ophthamology"
```

Comment:
Missing an `l` now, it's `OPHTHALMOLOGY`/Ophthalmology

Reply:
Either Github, or I am confused. But this seems fixed but is still blocking. So I am going to dismiss the review.